### PR TITLE
fix(ci): Change packages to have full versions and architectures

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net><!--

### CODE REVIEW

File: docs/reference/installation/_interfaces/rpm.cue [hunk 1]
```diff
@@ -20,7 +20,7 @@ installation: _interfaces: rpm: {
 	roles: [Name=string]: {
 		commands: roles._systemd_commands & {
 			_config_path: paths.config
-			install:      "sudo rpm -i https://packages.timber.io/vector/{version}/vector-{arch}.rpm"
+			install:      "sudo rpm -i https://packages.timber.io/vector/{version}/vector-{version}-1.{arch}.rpm"
```
Comment (binarylogic) [inline docs/reference/installation/_interfaces/rpm.cue:23]: Is the `-1.` intentional?

File: scripts/version.sh [hunk 2]
```diff
@@ -12,7 +12,4 @@ set -euo pipefail
 
 VERSION="${VERSION:-"$(sed -n 's/^version\s=\s"\(.*\)"/\1/p' Cargo.toml)"}"
 CHANNEL="${CHANNEL:-"$(scripts/release-channel.sh)"}"
-if [ "$CHANNEL" == "nightly" ]; then
-  VERSION="$VERSION-nightly"
-fi
```
Comment (binarylogic) [inline scripts/version.sh:17]: Is this intentional? If so, we should remove the `CHANNEL` var as well.

File: docs/reference/installation/_interfaces/rpm.cue (hunk 1 above)
Comment (jamtur01) [reply docs/reference/installation/_interfaces/rpm.cue:23]: Yep - RPM's have a default `-1.` suffixed to them.

File: scripts/version.sh (hunk 2 above)
Comment (jamtur01) [reply scripts/version.sh:17]: So ... I wish life was so simple. We use the `CHANNEL` environment variable in several places to direct packages to the night place.

**Comment by jamtur01** [comment]
This is definitely not elegant in places and there's a series of followup tickets we will build to refactor and replace some hacks we've added.

**Comment by jsoref** [comment]
Fwiw, this broke `act -l` https://github.com/nektos/act/issues/1595#issuecomment-1422713210

**Pull request merged**